morpheus/containers/impl/dynamic_matrix_impl.hpp

```diff
@@ -132,7 +133,7 @@ struct activate_impl {
   using variant   = typename MatrixFormats<Properties...>::variant;
   using type_list = typename MatrixFormats<Properties...>::type_list;
 
-  static void activate(variant &A, size_t idx) {
+  static void activate(const variant &A, size_t idx) {
```

Comment:
breaks down in switch_interface.cpp experiment. Need to check how to access the type list correctly for variant&

Reply:
Containers participating in variant weren't default constructible hence the issue.